### PR TITLE
tnftp 20230409

### PR DESCRIPTION
**Pull request by lukem**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/tnftp.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Tnftp < Formula
   homepage "https://cdn.netbsd.org/pub/NetBSD/misc/tnftp/"
   url "http://cdn.netbsd.org/pub/NetBSD/misc/tnftp/tnftp-20230409.tar.gz"
   sha256 "c6c631eb43cdeb5be47f8d37fa12b8d991894408b323d39ae753051f4271de2c"
-  license "BSD-4-Clause"
+  license "BSD-3-Clause"
```
Comment (chenrui333) [inline Formula/tnftp.rb:6]: Then this should be `license all_of: ["BSD-2-Clause", "BSD-3-Clause"]`

Comment (lukem) [reply Formula/tnftp.rb]: Thanks for the feedback. Fixed in commit [0a9a435](https://github.com/Homebrew/homebrew-core/pull/127923/commits/0a9a43588c09799d6e619a0508e790bfcf773ff7).

**Comment by chenrui333** [comment]
looks like just test failure on linux build at this point

**Comment by lukem** [comment]
> looks like just test failure on linux build at this point

Maybe a timeout in the test? Per the error logs:
```
  /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8_1/lib/ruby/2.6.0/timeout.rb:108:in `timeout'
  /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/test.rb:48:in `<main>'
```


Does the existing `tnftp.rb` formula on the master branch fail the same way?

A tnftp 20230409 build on CentOS 8 connects fine to the URI in the test:
```
./src/tnftp about:version
Version: tnftp 20230409

./src/tnftp ftp://anonymous:none@speedtest.tele2.net
Connected to speedtest.tele2.net.
220 Welcome to HGD FTP service.
331 Please specify the password.
230 Login successful.
Remote system type is UNIX.
Using binary mode to transfer files.
```

**Comment by lukem** [comment]
(can't see how to add the "help wanted" label through the UI)

**Comment by chenrui333** [comment]
> (can't see how to add the "help wanted" label through the UI)
you got it, meanwhile, can you also file an upstream issue for helping out this PR

**Comment by lukem** [comment]
I've released a newer version of tnftp - [20230507](https://ftp.netbsd.org/pub/NetBSD/misc/tnftp/tnftp-20230507.tar.gz).

Closing this pull request - a separate one can be opened for the newer version

**Comment by chenrui333** [comment]
seeing the same error with the new release, https://github.com/Homebrew/homebrew-core/pull/131051

I am gonna check out a bit tomorrow, if it just works fine on linux, maybe we can just skip the linux test.

**Pull request closed without merging**